src/Sarif.Driver/Sdk/RuleEnabledState.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk
```

Comment:
Nit:  Namespace should just be Driver

Reply:
This file lives in the Sdk  directory, so by convention s/be in that namespace. ?

---
In reply to: [169829799](https://github.com/Microsoft/sarif-sdk/pull/791#discussion_r169829799) [](ancestors = 169829799)